test/VoteManager.js

```diff
@@ -152,24 +139,28 @@ describe('VoteManager', function () {
         let epoch = await getEpoch();
         const stakerIdAcc3 = await stakeManager.stakerIds(signers[3].address);
         // const stakerIdAcc4 = await stakeManager.stakerIds(signers[4].address);
-        const staker = await stakeManager.getStaker(stakerIdAcc3);
+        let staker = await stakeManager.getStaker(stakerIdAcc3);
         const { biggestInfluencerId } = await getBiggestInfluenceAndId(stakeManager);
 
         const iteration = await getIteration(stakeManager, random, staker);
         await mineToNextState(); // propose
         await blockManager.connect(signers[3]).propose(epoch,
-          [1, 2, 3, 4, 5, 6, 7, 8, 9],
-          [100, 200, 300, 400, 500, 600, 700, 800, 900],
+          [],
```

Comment:
why was this modified?

Reply:
As this is the first epoch, there wont be any active assets. The require statement in propose will cause the transaction to fail if we follow the previous convention. Even if we create a collection in the first epoch, we have to go through an epoch wherein, an empty block is proposed and confirmed before that collection is activated and stakers start reporting as activation of newly created collection takes place when a block is confirmed

Reply:
then add the first epoch as a separate test in the beginning rather than modifying a bunch of tests which are testing various things. This can cause unknown bug to escape.